libraries/tarball.rb

```diff
@@ -29,6 +30,7 @@ def initialize(file_name, options = {})
       @reader = @data[1] =  Gem::Package::TarReader.new(Zlib::GzipReader.open @file_name)
       @prefix = options[:prefix]
       @options = options
+      @chown = !Gem.win_platform?
```

Comment:
For people coming at this code later it'd be nice if this had some doc on why it's used, and what it is doing. 

I'd also prefer it be something like `@can_chown`. That way the if checks read more like a sentence and are less ambiguous.